doc/api/tracing.md

```diff
@@ -23,15 +20,17 @@ The available categories are:
     measurements.
 * `v8`
 
-By default the `node`, `node.async_hooks`, and `v8` categories are enabled.
```

Comment:
Since `--trace-events-enabled` does not enable all categories we should still document this default list for that option.